File: 1975-minimum-distance-to-the-target-element/minimum-distance-to-the-target-element.py
class Solution(object):
    def getMinDistance(self, nums, target, start):
        """
        :type nums: List[int]
        :type target: int
        :type start: int
        :rtype: int
        """
        mn = float('inf')
        for i in range(len(nums)):
            if nums[i] == target:
                dist = abs(i-start)
                mn = min(mn,dist)
        return mn
        

File: 1975-minimum-distance-to-the-target-element/minimum-distance-to-the-target-element.cpp
class Solution {
public:
    int getMinDistance(vector<int>& nums, int target, int start) {
        int mn = INT_MAX;
        for(int i=0;i<nums.size();i++){
            if(nums[i]==target) {
            int dist = abs(i-start);
            mn = min(mn,dist);
            }
        }
        return mn;
    }
};